Hgt detection

kescobo opened the pull request:
`create_blast_db.py` now has function to create temporary fasta file from mongoDB and create a blast database.

Also removed import function that creates list of all records, but retained mongo import of list of records for later use.

kescobo committed:
begin analysis chunk

kescobo committed:
begin analysis chunk

kescobo committed:
Import one record at a time, begin blast analysis

kescobo committed:
Import one record at a time, begin blast analysis

kescobo committed:
Merge branch 'master' of github.com:kescobo/kvasir into hgt_detection

kescobo committed:
Create blast db, fix settigs and mongo import

kescobo committed:
Create blast db, fix settigs and mongo import

kescobo committed:
Create blast db, fix settigs and mongo import

kescobo committed:
create temporary fasta within `db_cds_to_fna` instead

kescobo committed:
create temporary fasta within `db_cds_to_fna` instead

kescobo committed:
create temporary fasta within `db_cds_to_fna` instead

kescobo committed:
create temporary fasta within `db_cds_to_fna` instead

lydiaauch commented on src/DataImport/mongo_import.py:
```diff
@@ -5,12 +5,21 @@
+    :param record: gene record
+    :param collection: MongoDB collection name
+    """
+    db[collection].insert_one(record)
```
Add newline at end of file

kescobo replied:
Done.

kescobo committed:
Add Pycharm config file (.idea)

kescobo committed:
Add Pycharm config file (.idea)

kescobo committed:
Add Pycharm config file (.idea)

kescobo committed:
Add Pycharm config file (.idea)

kescobo committed:
Add newlines to ends of files

kescobo committed:
Add newlines to ends of files

kescobo committed:
Add newlines to ends of files

lydiaauch committed:
create smaller functions in gb_parse

kescobo committed:
Continuing work on blasting

kescobo committed:
Continuing work on blasting

kescobo committed:
Continuing work on blasting

kescobo committed:
Continuing work on blasting

kescobo committed:
Merge branch 'import_fix'

kescobo committed:
Merge branch 'master' into hgt_detection

# Conflicts:
#	run.py
#	src/DataImport/gb_parse.py
#	src/DataImport/mongo_import.py
#	src/FindHGT/create_fasta.py

kescobo committed:
Change usage of `MONGODB`, continue blast result import

kescobo committed:
Change usage of `MONGODB`, continue blast result import

kescobo committed:
Merge remote-tracking branch 'origin/hgt_detection' into hgt_detection

# Conflicts:
#	.idea/workspace.xml
#	run.py
#	settings.py
#	src/DataImport/gb_parse.py
#	src/FindHGT/make_blast_db.py
#	src/FindHGT/run_blast.py

kescobo committed:
Change usage of `MONGODB`, continue blast result import

kescobo committed:
Change usage of `MONGODB`, continue blast result import

kescobo committed:
Change usage of `MONGODB`, continue blast result import

Import one record at a time, begin blast analysis

Create blast db, fix settigs and mongo import

kescobo committed:
I have no idea...

kescobo committed:
Rebased onto import_generator branch

kescobo committed:
Rebased onto import_generator branch

kescobo committed:
Rebased onto import_generator branch

kescobo commented:
@lydiaauch If you have no objection, I think I want to close this, squash a bunch of the commits and then open a new one - this is super messy,

kescobo commented:
Closing to clean up the mess I made.
